blablas/ep24/index.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2019-12-29
+time: 21h
+duration: "2:18:03"
+title: "Geeksblabla MSS with Yassine El Kachchani"
+tags: ["startup", "side hustle", "morocco"]
+isNext: false
+video: "2770524023008256"
+published: true
+url: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+podcastUrl: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+---
+
+In this episode of GeeksBlabla, we discuss startuping with Yassine, the CEO of United Remote.
+
+## Guests
+
+[Yassine El Kachchani](https://www.facebook.com/kachchani)
+
+[Mohammed Aboullaite](https://www.facebook.com/aboullaite)
+
+## Notes
+
+If you want contribute in Geksblabla project by adding the notes check [this issue](https://github.com/DevC-Casa/geeksblabla.com/issues/23/)
+
+0:03 - Who is Yassine El Kachchani ?
```

Comment:
Make sure to update times to correct format `hh:mm` ex: `00:03`

Reply:
[This issue](https://github.com/DevC-Casa/geeksblabla.com/issues/23) mentions "h:min" in the #Notes section though, is that cool ?

Reply:
Hmmm, ok.